FakeItEasy.sln

```diff
@@ -1,7 +1,7 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0
-MinimumVisualStudioVersion = 10.0.40219.1
+# Visual Studio 14

```

Comment:
Is it called "14" or should we have VS2015 here? Or did the tools choose this name?